7a_temp_coop_munge/src/data_parsers/parse_wilter_files.R

```diff
@@ -17,8 +20,24 @@ parse_long_term_ntl <- function(inind, outind) {
   outfile <- as_data_file(outind)
   raw_file <- data.table::fread(infile, select = c("lakeid", "sampledate",
                                                    "depth", "wtemp"))
-  clean <- raw_file %>% rename(UWID = lakeid, DateTime = sampledate, Depth = depth,
-                               temp = wtemp) %>% mutate(DateTime = as.Date(DateTime))
+  clean <- raw_file %>%
+    rename(UWID = lakeid, DateTime = sampledate,
+           Depth = depth, temp = wtemp) %>%
+    mutate(DateTime = as.Date(DateTime)) %>%
+    mutate(WBIC = case_when(
+      UWID == 'AL' ~ '2332400',
+      UWID == 'BM' ~ '1835300',
+      UWID == 'CB' ~ '2017500',
+      UWID == 'CR' ~ '1842400',
+      UWID == 'FI' ~ '985100',
+      UWID == 'ME' ~ '805400',
+      UWID == 'MO' ~ '804600',
+      UWID == 'SP' ~ '1881900',
+      UWID == 'TB' ~ '2014700',
+      UWID == 'TR' ~ '2331600',
+      UWID == 'WI' ~ '805000')) %>%
```

Comment:
Would it be useful to pull this table out as a crosswalk file, in case it's needed somewhere else?